subsys/net/l2/openthread/openthread.c

```diff
@@ -75,6 +74,54 @@ static struct net_linkaddr *ll_addr;
 
 static struct net_mgmt_event_callback ip6_addr_cb;
 
+static struct k_poll_signal openthread_signals[OPENTHREAD_SIGNALS];
+static struct k_poll_event openthread_events[OPENTHREAD_SIGNALS];
+
+
+bool ot_signal_check_clear(enum openthread_signal_id signal_id,
+						int *signal_value)
```

Comment:
wrong indentation. It should be below last '(' + 1 space. (linux code style).
And if you don't know, just look at the file, originally. 

Apply that everywhere